.lfsconfig

```diff
@@ -1,3 +1,3 @@
 [lfs]
-	url = https://git-lfs.lsst.cloud/lsst/ap_verify_ci_dc2
+	url = https://git-lfs-rw.lsst.cloud/lsst/ap_verify_ci_dc2
```

Comment:
While this does work for me, won't it mean that anyone who wants to download a dataset (including Jenkins) now needs to set up push access, which is otherwise only needed for active developers?

Also, this should really be a separate commit from the changes to the dataset contents.

Reply:
Good catch--didn't intend to commit this, I'll back it out.